openlibrary/macros/SearchResultsWork.html

```diff
@@ -59,7 +59,7 @@ <h3 class="booktitle">
               <span class="preview-covers">
                 $for x, i in enumerate(doc.get('ia')[1:10]):
                   <a href="$(book_url)?edition=$(i)">
-                    <img width="30" height="45" loading="lazy" src="//archive.org/download/$(i)/page/cover_w60_h60.jpg" alt>
+                    <img width="30" height="45" loading="lazy" src="//archive.org/download/$(i)/page/cover_w60_h60.jpg" alt="Cover of edition $i">
```

Comment:
Ideally, this would be more meaningful e.g. describing published version but I don't think we have that information
e.g. this is an example of a cover of $name published in 2010` as "Cover of edition 1" is not very descriptive.

I think "Cover of $bookname ($1)" might be better than this, but this is better than nothing :)

Reply:
Why is more English-only text being added to the site? How does this improve accessibility for non-English speakers?

Reply:
I did write this in my review and suggested a follow-up to fix this. Accessibility is just as important and this is an improvement there.

Reply:
@nynaalekhya please make sure to follow up here as well as #4735 on translating these per Tom's comment. Happy to explain how i18n works in Slack if you need any help with that, but thanks for taking care of the accessibility side.

Reply:
@jdlrobson I will make sure to include the changes in messages.pot file. I'll message you on Slack if I have any further queries. 
Thank you :)

Reply:
Hi @jdlrobson,
Could you invite me to the slack channel 
email: nynalaalekhya@gmail.com